public/app/features/alerting/unified/components/rules/RulesFilter.tsx

```diff
@@ -99,7 +104,6 @@ const RulesFilter = ({ onFilterCleared = () => undefined }: RulesFilerProps) =>
   const handleAlertStateChange = (value: PromAlertingRuleState) => {
     logInfo(LogMessages.clickingAlertStateFilters);
     updateFilters({ ...filterState, ruleState: value });
-    setFilterKey((key) => key + 1);
```

Comment:
why are we removing this ` setFilterKey((key) => key + 1);` ??

Reply:
Not needed for the state change filter